python/ray/autoscaler/_private/load_metrics.py

```diff
@@ -99,12 +99,6 @@ def prune(mapping):
         prune(self.resource_load_by_ip)
         prune(self.last_heartbeat_time_by_ip)
 
-    def approx_workers_used(self):
-        return self._info()["NumNodesUsed"]
-
-    def num_workers_connected(self):
-        return self._info()["NumNodesConnected"]
-
```

Comment:
You should also be able to remove the fields from `_info` (and the associated calculations) along with `_get_resource_usage`,